chia/cmds/coin_funcs.py

```diff
@@ -44,6 +45,10 @@ async def async_list(args: Dict[str, Any], wallet_client: WalletRpcClient, finge
         excluded_amounts=final_excluded_amounts,
         excluded_coin_ids=excluded_coin_ids,
     )
+    coin_type: str = "XCH" if addr_prefix == "xch" else "TXCH"
+    if wallet_type == WalletType.CAT:
+        coin_type = "CAT"  # override XCH/TXCH prefix for CAT wallet
```

Comment:
https://github.com/Chia-Network/chia-blockchain/blob/d13f8b837fd585c122b4c7366bf723aea1d4affd/chia/cmds/show_funcs.py#L131

I'd probably prefer an `if`/`elif`/`else` over picking the distinctly wrong thing first.

```suggestion
    if wallet_type == WalletType.XCH:
        coin_type: str = "XCH" if addr_prefix == "xch" else "TXCH"
    elif wallet_type == WalletType.CAT:
        coin_type = "CAT"  # override XCH/TXCH prefix for CAT wallet
    else:
        <something, i dunno what we want to do>
```

This is also relevant I think.

https://github.com/Chia-Network/chia-blockchain/blob/d13f8b837fd585c122b4c7366bf723aea1d4affd/chia/cmds/wallet_funcs.py#L87-L100

Reply:
yea, I worried about getting the entire CAT name to display here. since it could be quite long to display in each coin

Reply:
This is also why I went with the simpler form of naming it first - it should be XCH/TXCH in most cases, except only for CAT wallets so an alternative would be 
```
    if wallet_type == WalletType.CAT:
        coin_type = "CAT"  # override XCH/TXCH prefix for CAT wallet
    else 
        coin_type: str = "XCH" if addr_prefix == "xch" else "TXCH"
```

Reply:
changed